tests/test_fields.py

```diff
@@ -2016,6 +2017,16 @@ def test_allow_null(self):
         output = field.run_validation(None)
         assert output is None
 
+    def test_allow_empty(self):
```

Comment:
I would change the name to `test_allow_empty_disallowed` (or similar), as `test_allow_empty` implies `allow_empty=True`.

Reply:
Ok, renamed